sources/models/defaults.toml

```diff
@@ -77,6 +77,15 @@ restart-commands = ["/usr/bin/host-containers"]
 [metadata.settings.host-containers]
 affected-services = ["host-containers"]
 
+# Network
+
+[configuration-files.proxy-env]
+path = "/etc/shared/services/network/proxy/env"
```

Comment:
This seems really deep.  Can we simplify to `/etc/network/proxy.env` or something?